packages/common/services/console-logger.service.ts

```
@@ -40,22 +108,54 @@ const dateTimeFormatter = new Intl.DateTimeFormat(undefined, {
+    contextOrOptions?: string | ConsoleLoggerOptions,
     @Optional()
-    protected options: ConsoleLoggerOptions = {},
+    options?: ConsoleLoggerOptions,
   ) {
-    if (!options.logLevels) {
-      options.logLevels = DEFAULT_LOG_LEVELS;
-    }
+    // eslint-disable-next-line prefer-const
+    let [context, opts] = isString(contextOrOptions)
+      ? [contextOrOptions, options]
+      : !!options
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)